src/main/java/pl/pop/interview/master/question/QuestionServiceException.java

```diff
@@ -0,0 +1,7 @@
+package pl.pop.interview.master.question;
+
+public class QuestionServiceException extends RuntimeException{
```

Comment:
QuestionsServiceException?

Dlaczego tak to jest nazwane?

Reply:
Domyślam się, że chodzi maksymalnie ogólną nazwę na wyjątek logiki biznesowej, wymagający doprecyzowania w message. Myślę, że może lepszą nazwą byłoby "QuestionException". Ja podpowiadam to podejście jako robocze rozwiązanie na moich lekcjach zanim dojdziemy do sterowania statusami odpowiedzi, andotacji walidacji itp.